src/components/ArticleIndividual/CommentsList.jsx

```diff
@@ -16,6 +18,39 @@ function CommentsList({ article_id }) {
         setLoading(false);
       });
   }, []);
+  function handlePostTextInput(event) {
+    console.log(event.target.value, "new comment to post");
+    setCurrentNewComment(event.target.value);
+  }
+  function handleSubmitCommentButton(event) {
+    console.log(currentNewComment, "attempted to post this");
+    console.log(article_id, "<--- article id");
+    setError(null); // set errror to null
+
+    console.log(currentNewComment, "useState() new comment to post");
+
+    // post request
+    apiClient
+      .post(`/api/articles/${article_id}/comments`, {
+        username: "jessjelly",
```

Comment:
absolutely fine to hard code the user, but make it clear to the person using the website that they will comment as this username